grails-app/services/org/pih/warehouse/shipping/ShipmentService.groovy

```diff
@@ -2376,4 +2376,11 @@ class ShipmentService {
         }
         return referenceNumber
     }
+
+    Comment addShipmentComment(Shipment shipment, String content, User sender, User recipient) {
```

Comment:
I don't mind it, but probably according to our code conventions, we shouldn't have more than 3 arguments (?) :thinking: 
I believe we need to figure out a way how to deal with such cases (command object??)